src/server/json_family.cc

```diff
@@ -1447,7 +1447,11 @@ io::Result<JsonPathV2, string> ParsePathV2(string_view path) {
   }
 
   if (absl::GetFlag(FLAGS_jsonpathv2)) {
-    return json::ParsePath(path);
+    auto path_result = json::ParsePath(path);
```

Comment:
`ParsePath` is also used in `JsonAccessor::GetPath` so maybe it's worth checking if you can return `kSyntaxErr` directly from `ParsePath`. I am not sure how this affects `GetPath` but if it doesn't we don't need to transform the error here (that is, we can just return `json::ParsePath(path))`

Reply:
We can just move the `LOG(WARNING) << "Invalid Json path: " << field << ' ' << ec_msg;` to `json::ParsePath` . Furthermore we can add an extra `LOG(WARNING)` to `MakeJsonPathExpr`. 

This is nice because now we got logs for other paths as well :) 

I am preacceppting your PR since this is not mandatory but I find it a good addition. If you don't like it I am not pushing this :)